matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml

```diff
@@ -156,14 +161,17 @@ $xwiki.jsx.use('PhenoTips.Widgets')##
                       #set ($serverName = $serverID)
                     #end
                     #set ($disclaimer = $config.getProperty('disclaimer').getValue())
-                    #set ($matchesUpdateObject = $matchesRunDoc.getObject($infoClassname, 'serverName', $serverID))
-                    #if ($matchesUpdateObject)
-                      #set($endTime = $matchesUpdateObject.getValue('completedTime'))
+                    #set ($endTime = "$!matchesRunDoc.getObject($infoClassname, 'serverName', $serverID).getValue('completedTime')")
```

Comment:
Now that you've moved the message generation from JS to Velocity, a different approach might be more appropriate, to avoid re-parsing the date from a string:

```
#set($endTime = '')## makes sure the previous value is erased
#set ($endTime = $!matchesRunDoc.getObject($infoClassname, 'serverName', $serverID).getValue('completedTime'))## stores the new value IF it exists
```

Then you can check if the date exists with:
```
#if ("$!endTime" != "")
````